crates/cargo-remove/tests/testsuite/cargo_remove/remove-basic.in/Cargo.toml

```diff
@@ -11,16 +11,14 @@ semver = "0.1.0"
 
 [dependencies]
 docopt = "0.6"
-pad = "0.1"
-rustc-serialize = "0.3"
+rustc-serialize = "0.4"
 semver = "0.1"
 toml = "0.1"
-clippy = {git = "https://github.com/Manishearth/rust-clippy.git", optional = true}
```

Comment:
I'm guessing this should instead be mocked with a fake git repo. I'll make that change with fresh eyes in the morning. :)

Reply:
I can't seem to comment on the lines directly since no changes were made, but there are also a bunch of unused mock packages that I understand to be copied over from cargo add (e.g. `linked-hash-map`). Should those be removed?

Reply:
Yes, it'll  make the tests faster to run

Reply:
Okay, in the interest of not expanding the scope of this PR, I think the following changes should be made in a subsequent PR:
- Add test for removing git deps with the help of `cargo_test_support::git`
- Clean up unused testing infrastructure (per my last comment)
- Rename tests to be like cargo add, without `remove` in the name

Reply:
Sounds good